packages/live-share-inking/src/input/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+export * from "./InputProvider";
```

Comment:
nit: alphabetize

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512